### DISCUSSION TITLE
What is the longest 🚂 tunnel in the world?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#799] [Galaxy Trivia] [answered]
**Bonus:** How many Great Pyramids of Giza can you build with the rocks removed from the tunnel?

**Comment by peterbaccaro** [comment, marked as answer]
World's longest railway tunnel, **Gotthard Base Tunnel, Switzerland**. It is an extraordinary feat of engineering, stretching approximately 57 kilometers (35.4 miles) beneath the Swiss Alps. The Gotthard Base Tunnel officially opened in June 2016 and provides a vital link between northern and southern Europe, allowing for faster and more efficient train travel.

Amount of excavated rock: 28,200,000 t, 13,300,000 m3 or 17,400,000 cu yd, the equivalent of **5 Giza pyramids**.

https://www.guinnessworldrecords.com/world-records/longest-rail-tunnel

**Comment by SANKARAMDAS** [comment]
Gotthard Base Tunnel, Switzerland
The Gotthard Base Tunnel is the world's longest and deepest tunnel. It runs under the Swiss Alps between the towns of Erstfeld in the north and Bodio in the south. The tunnel is 57 km long and reaches a depth of 2,300 meters.  Trains reaching speeds of up to 250 kilometers an hour can travel through in 20 minutes, according to the Swiss Travel System. The tunnel has helped to decrease the travel time between Zurich, Switzerland, and Milan, Italy, by an hour.

The total amount of material(rocks) 17,900,000 cu yd or 28,000,000 t, 14,000,000 m3, the equivalent of 5 and 1/2 Giza pyramids.
